google/cloud/bigtable/benchmarks/setup.cc

```diff
@@ -17,19 +17,20 @@
 #include "google/cloud/internal/build_info.h"
 #include "google/cloud/internal/random.h"
 #include "google/cloud/internal/throw_delegate.h"
+#include <algorithm>
 #include <cctype>
-#include <iomanip>
+#include <ctime>
 #include <iostream>
 #include <sstream>
 
 /// Supporting types and functions to implement `BenchmarkSetup`
 namespace {
 std::string FormattedStartTime() {
-  auto start = std::chrono::system_clock::now();
-  std::time_t start_c = std::chrono::system_clock::to_time_t(start);
+  auto const start = std::chrono::system_clock::now();
+  auto const start_as_time_t = std::chrono::system_clock::to_time_t(start);
```

Comment:
Are these edits related to the common version jump?

Reply:
Not really, let me revert them.